### PR TITLE
Bookings: Add isPastBooking field

### PR DESCRIPTION
**Pull request by tbergq**
I know there is PAST | FUTURE field when querying allBookings, but this is useful when querying booking(id). 

There are different things to be done on client when a flight is past vs future. 
It makes sense to be able to access this property easily.

### CODE REVIEW

File: src/booking/queries/__tests__/AllBookings.isPastBooking.test.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    `${allBookings}/2707224\\?simple_token=[0-9a-f-]{36}`,
+  ).replyWithData(Booking2707224Dataset);
+});
+
+it('should return isPastBooking', async () => {
+  const query = `{
+      allBookings {
+        edges {
+          node {
+            isPastBooking
```
Comment (mrtnzlml) [inline src/booking/queries/__tests__/AllBookings.isPastBooking.test.js]: @tbergq Is your intention to test here the return value or are you just making sure the query is fine (will work in the future)?

Comment (tbergq) [reply src/booking/queries/__tests__/AllBookings.isPastBooking.test.js]: Making sure the query is fine.

There are other tests for the isBooking function.

Comment (mrtnzlml) [reply src/booking/queries/__tests__/AllBookings.isPastBooking.test.js]: You can just use `validate( ... )` to make sure the query is not invalid. But I will accept it.

Comment (tbergq) [reply src/booking/queries/__tests__/AllBookings.isPastBooking.test.js]: Ok, I will definitely use that next time 🙂

**Pull request merged**